src/data/i18n.tsx

```diff
@@ -19,3 +19,20 @@ export const getTranslation: <T = any>(
 export const getTranslations: <T = any>(
   data: TranslationCollection<T>,
 ) => WithLocale<T>[] = R.prop('translations')
+
+export const getTranslationRecursively: any = (locale: Locale) =>
```

Comment:
Is it possible to type this with generics?

Reply:
If not you could just change it to 
```suggestion
export const getTranslationRecursively : (locale: Locale) => (multilingual : any) => any =
```
for now. Or maybe

```suggestion
export const getTranslationRecursively : <T=any, S=any>(locale: Locale) => (multilingual : T) => S =
```
(though that isn't really any better than my previous suggestion...)

Reply:
It might be possible with recursive type references which are supported since typescript 3.7.0, see: https://github.com/microsoft/TypeScript/pull/33050